test/slow/api_spec_slow.js

```diff
@@ -156,6 +157,20 @@ describe(`electron-forge API (with installer=${installer.substr(12)})`, () => {
     });
   });
 
+  describe('import', () => {
+    before(async () => {
+      await ensureTestDirIsNonexistent();
+      await fs.mkdir(dir);
+      execSync(`${installer} init -y`, {
+        cwd: dir,
+      });
```

Comment:
This'll need to change to use the API instead of shelling out, that seems wrong.

Reply:
Which API are you referring to? I didn't use `yarnOrNpm()` because then it wouldn't respect a command-line override ala [this](https://github.com/electron-userland/electron-forge/blob/master/test/slow/api_spec_slow.js#L15) line.

It kinda makes sense to me to shell out because this is the equivalent of the user creating a project themselves in their command-line shell before ever firing up `electron-forge`, but that's kinda fuzzy thinking, and not something I feel strongly about.

So if you still want this changed and can to clarify how you'd like this done, I can make the change.

Reply:
I was referring to using the `init` API:

https://github.com/electron-userland/electron-forge/blob/1b69e1b84db9cca5273f90f5cc9c3fc58843ce25/src/api/init.js#L23-L29

Reply:
Oh, that's not the use case I was aiming for. My intent here was to set up a vanilla node project using `yarn init -y` or `npm init -y` so that the `forge.import()` (called via API in the test case itself) would be run on a new un-forgified project. I thought that was how the `import` operation is meant to be used, and that's as close as I can reasonably get to testing the `ember-electron` use case where somebody installs it into an already-existing Ember project that doesn't have any electron or `forge` or anything, and then the `ember-electron` install process runs `forge import`. Does that make sense?

Reply:
Ah, I totally misread that code. That variable needs to be called `nodeInstaller` or something.